src/templates/XmlRenderer/template.tsx

```diff
@@ -0,0 +1,967 @@
+import React, { FunctionComponent } from "react";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import styled from "@emotion/styled";
+import { XmlRendererFileInterface } from "./types_file";
+import { XmlRendererInterface } from "./types_data";
+import { processors } from "xml2js";
+
+const TemplateContainer = styled.div`
+  .template-container {
+    width: 210mm;
+    height: 297mm;
+    display: block;
+    position: relative;
+    background-color: white;
+    color: black;
+    font-family: "Arial";
+    font-size: 12pt;
+    padding: 35px 70px;
+    margin: auto;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+  }
+
+  .divTable {
+    display: table;
+    width: 100%;
+  }
+  .divTableRow {
+    display: table-row;
+  }
+  .divTableHeading {
+    background-color: #eee;
+    display: table-header-group;
+  }
+  .divTableCell,
+  .divTableHead {
+    border: 1px solid #999999;
+    display: table-cell;
+    padding: 1fr 3fr;
+  }
+  .divTableHeading {
+    background-color: #eee;
+    display: table-header-group;
+    font-weight: bold;
+  }
+  .divTableFoot {
+    background-color: #eee;
+    display: table-footer-group;
+    font-weight: bold;
+  }
+  .divTableBody {
+    display: table-row-group;
+  }
+
+  @media print {
+    .template-container {
+      box-shadow: none;
+      margin: 0;
+      border: none;
+    }
+  }
+
+  @page {
+    margin: 0;
+  }
+`;
+
+export const XMLRenderer: FunctionComponent<TemplateProps<XmlRendererFileInterface> & {
+  classNameName?: string;
+}> = ({ document }) => {
+  // Decode html entities and remove byte order mark
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const entitiesDecoder = require("html-entities-decoder");
+  const output = entitiesDecoder(document.xmlData).replace("\\ufeff", "");
+
+  // Parse XML to Json
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const xml2js = require("xml2js");
```

Comment:
why require syntax?
can you do a top level import?
import { Parser } from xml2js (line 6)

same for html-entitties-decoder as well (also why did you choose to use this package?)

Reply:
Done